src/networks.ts

```diff
@@ -1103,17 +1231,38 @@ export const networks: Networks = {
     key: 'ethereumTestnetGoerli',
     shortKey: 'goerli',
     color: '#83EEFF',
-    chain: 5,
+    chain: 11155111,
     rpc: process.env.ETHEREUM_TESTNET_GOERLI_RPC_URL || 'https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
     holographId: 4000000011,
     tokenName: 'Ethereum Goerli',
     tokenSymbol: 'ETH',
     explorer: 'https://goerli.etherscan.io',
     lzEndpoint: '0xbfD2135BFfbb0B5378b56643c2Df8a87552Bfa23'.toLowerCase(),
     lzId: 10121,
+    ccipEndpoint: '0x0000000000000000000000000000000000000000'.toLowerCase(),
+    ccipId: 0,
     active: true,
     protocolMultisig: '0xcb5E71E000288e2648Bb0E7b67D3f2850e56EF67'.toLowerCase(),
   } as Network,
+  ethereumTestnetSepolia: {
+    name: 'Ethereum Sepolia',
+    type: NetworkType.testnet,
+    key: 'ethereumTestnetSepolia',
+    shortKey: 'sepolia',
+    color: '#83EEFF',
+    chain: 5,
+    rpc: process.env.ETHEREUM_TESTNET_GOERLI_RPC_URL || 'https://eth-sepolia.g.alchemy.com/v2/demo',
+    holographId: 4000000011,
+    tokenName: 'Ethereum Sepolia',
+    tokenSymbol: 'ETH',
+    explorer: 'https://sepolia.etherscan.io',
+    lzEndpoint: '0x7cacBe439EaD55fa1c22790330b12835c6884a91'.toLowerCase(),
+    lzId: 161,
+    ccipEndpoint: '0xD0daae2231E9CB96b94C8512223533293C3693Bf'.toLowerCase(),
+    ccipId: '16015286601757825753',
+    active: true,
```

Comment:
@alexanderattar this should probably still be false. Otherwise things like the interface will get updated when we do a deployment.

Reply:
Does the frontend display Sepolia if this is set to active already? I think we need start thinking about about deprecating Goerli and moving to Sepolia anyways, and I'd like to at least be able to start using this for the protocol and CLI. Our Goerli ETH supply is getting low. Also many RPC providers including Alchemy are deprecating Goerli by Jan 2024. CCIP doesn't support Goerli